### PR TITLE
Create libremesh-automation_monitoring.md

### PR DESCRIPTION
**Pull request by digitigrafo**
LibreMesh GSoC2023 project for automation and monitoring

### CODE REVIEW

File: collections/_projects/libremesh-automation_monitoring.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ - "Ansible"
+ - "OpenWrt"
+ - "lua, bash"
+ - "publish code and doc"
+ - "report the process"
+difficulty: "medium"
+size: "350 hours"
+issues:
+ - "https://github.com/coala/coala/issues/####"
+mentors:
```
Comment (andibraeu) [inline collections/_projects/libremesh-automation_monitoring.md:17]: can you please use github usernames for mentors? Links are created based on that.

**Review by andibraeu — CHANGES_REQUESTED** [review]

File: collections/_projects/libremesh-automation_monitoring.md (hunk 1 above)
Comment (ilario) [reply collections/_projects/libremesh-automation_monitoring.md:17]: Uh, we didn't know that :)
My username on Github is `ilario`

Comment (ilario) [reply collections/_projects/libremesh-automation_monitoring.md:17]: Oh, @digitigrafo already fixed, thanks!

Comment (digitigrafo) [reply collections/_projects/libremesh-automation_monitoring.md:17]: yes, mine digitigrafo, I just commit in patch-1

**Comment by andibraeu** [comment]
great! thank you!

**Pull request merged**